bids/modeling/hrf.py

```diff
@@ -309,7 +309,7 @@ def _orthogonalize(X):
     """
     if X.size == X.shape[0]:
         return X
-    from scipy.linalg import pinv
+    from numpy.linalg import pinv
     for i in range(1, X.shape[1]):
         X[:, i] -= np.dot(np.dot(X[:, i], X[:, :i]), pinv(X[:, :i]))
```

Comment:
Alternative 1: If indeed it is an efficiency problem when `n_rows > n_cols`, we can simply transpose before and after `pinv`.

```suggestion
    from scipy.linalg import pinv
    for i in range(1, X.shape[1]):
        # Pseudoinverse commutes with transpose, and is more efficient if # rows < # cols
        X[:, i] -= np.dot(np.dot(X[:, i], X[:, :i]), pinv(X[:, :i].T).T)
```